spec/reservation_repo_spec.rb

```diff
@@ -0,0 +1,98 @@
+require_relative '../lib/reservation_repository'
+require_relative '../lib/user_repository'
+require_relative '../lib/space_repository'
+
+def reset_tables
+  sql_seed = File.read('/Users/paris/Desktop/Projects/makersbnb/spec/seeds/makers_bnb_seed.sql')
```

Comment:
this needs to be changed to 'spec/seeds/makers_bnb_seed.sql' so we can all use it !

Reply:
Oh yes ofc